docs/docs/hpc-on-k8s/genome.md

```diff
@@ -0,0 +1,62 @@
+# Genome Workload
+FastQC is a widely used tool for assessing the quality of genome sequence data. It provides a comprehensive overview of various quality metrics that are crucial for ensuring the reliability and accuracy of genomic data. 
+
+
+## Steps to Execute Job
+Below is the example manifest file for performing genome quality checks in a Kubernetes cluster.
+```yaml
+# job.yaml
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: fastqc-
+  namespace: default
```

Comment:
As discussed IRL I think it's worth this job using the `bio-context` otherwise the `users` section isn't being put to any use